### ISSUE TITLE
`buffrs install` fails when the `proto/vendor/`directory does not exist

### ISSUE THREAD

**Issue by qsantos** [helsing-ai/buffrs#192]
```

**Comment by qsantos** [comment]
Putting aside #192, it looks like the intended behavior is indeed to fail to run when `proto/` does not exist:

https://github.com/helsing-ai/buffrs/blob/4a6d0e9084638f40909052aa1894bc20facbc01b/src/package/store.rs#L58-L67

So the question behind this issue is now: should the `proto/` directory be created automatically?

**Comment by mara-schulke** [comment]
Yes, but in `install` we should check for existence and initialize it if it doesn't exist.
